FrontEnd/src/components/Navbar.tsx

```diff
@@ -1,23 +1,61 @@
-import * as React from 'react';
 import AppBar from '@mui/material/AppBar';
 import Box from '@mui/material/Box';
 import Toolbar from '@mui/material/Toolbar';
 import IconButton from '@mui/material/IconButton';
 import AccountCircle from '@mui/icons-material/AccountCircle';
 import SelectedMenu from './SelectedMenu';
+import { useNavigate } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import Link from '@mui/material/Link';
+import { Typography } from '@mui/material';
 
 export default function Navbar() {
+  let navigate = useNavigate();
+  const [userInfo, setUserInfo] = useState<any>();
```

Comment:
Instead of `any` we should define a TypeScript interface that includes the data that can be in the `userInfo` object. To handle the fact that that the result can be undefined the type would be nullable, see: https://www.typescriptlang.org/docs/handbook/advanced-types.html#nullable-types

Reply:
Great point, thank you @OmarShehata! I will fix it in the next PR